packages/dx-react-chart/src/utils/series-helper.jsx

```diff
@@ -15,15 +15,28 @@ import {
 export const declareSeries = (pluginName, { components, getPointTransformer, createHitTester }) => {
   class Component extends React.PureComponent {
     render() {
-      const { name } = this.props;
+      const {
+        name,
+        argumentField,
+        valueField,
+        scaleName,
+        seriesComponent,
+        pointComponent,
+        color,
+        ...restProps
+      } = this.props;
       const symbolName = Symbol(name);
       const seriesItem = {
         getPointTransformer,
         createHitTester,
         ...this.props,
```

Comment:
`seriesItem` contains common series configuration - specific and appearance settings are merged later in `addSeries`.
common configuration - settings separated from `restProps` several lines above.